internal/pkg/template/templates/workloads/services/lb-web/manifest.yml

```diff
@@ -32,7 +32,11 @@ memory: {{.Memory}}    # Amount of memory in MiB used by the task.
 platform: {{.Platform}}
 {{- end}}
 count: {{.Count.Value}}       # Number of tasks that should be running in your service.
+{{- if not .Platform}}
 exec: true     # Enable running commands in your container.
+{{- else if ne .Platform.PlatformString "windows/amd64"}}
```

Comment:
Instead of hard-coding the string what do you think of making the `isWindows` method of `*TaskConfig` public so that we can do?

```go
{{- if not .IsWindows() }}
exec: true
{{- end }}
```